src/main/java/org/wildfly/extension/elytron/SSLDefinitions.java

```diff
@@ -136,8 +138,8 @@
             .setAllowExpression(true)
             .setMinSize(1)
             .setAllowNull(true)
-            .setAllowedValues(allowedValues(Protocol.values()))
-            .setValidator(new EnumValidator<>(Protocol.class, false, true))
+            .setAllowedValues(Protocol.names())
```

Comment:
As I mention in the other PR, I think depending on Elytron for the list of acceptable values is going to be a problem, if Elytron us updated this list will change which will really mean we should bump the model version so we can handle it with transofrmers.

I think really we are just going to have to hard code the values in the model.